commands/General/Role Manipulation/toggleroles.js

```diff
@@ -0,0 +1,33 @@
+const { Command } = require('klasa');
+
+/* You need to add the following lines to your guild schematic manager:
+     	.add('selfroles', folder => folder
+		.add('roles', 'role', { array: true })
+		.add('enabled', 'boolean', { default: true }))
+*/
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			runIn: ['text'],
+			cooldown: 10,
+			aliases: ['toggleselfrole', 'enableselfroles', 'disableselfroles'],
+			permissionLevel: 6,
+			requiredPermissions: ['USE_EXTERNAL_EMOJIS'],
+			description: language => language.get('COMMAND_TOGGLE_SELFROLES'),
+			extendedHelp: 'No extended help available.'
+		});
+	}
+
+	async run(msg) {
+		const current = msg.guild.settings.get('selfroles.enabled');
+		msg.guild.settings.update('selfroles.enabled', !current);
+		if (current) {
+			return msg.send(`｢ **Self Roles** ｣ Self Roles have been globally disabled. Happy Hunting for Inertia!`);
+		} else {
```

Comment:
Useless else

Reply:
I'd be tempted to do
```js
return msg.send(`｢ **Self Roles** ｣ Self Roles have been globally ${current ?
    'disabled. Happy Hunting for Inertia!' :
    "enabled. Seems like we got 'em! :)"}`);
```

Reply:
Also there's an inconsistency here: in one place they're referred to in the plural, in another place, in the singular.